### PR TITLE
dev-python/graphyte: initial ebuild for version 1.6.0

### PR DESCRIPTION
**Pull request by 65278**
Graphyte ebuild with tests. A simple library to send data to graphite, no dependencies.

### CODE REVIEW

File: dev-python/graphyte/graphyte-1.6.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+{
+	einfo "Testing graphyte."
+	python_test()
+	{
+		$PYTHON test_graphyte.py --verbose
+	}
+	if ! python_foreach_impl python_test
+	then
+			eerror "Python tests failed!"
+			die
```
Comment (alip) [inline dev-python/graphyte/graphyte-1.6.0.ebuild:27]: nitpick: you can shorten this to `python_foreach_impl python_test || die "Python tests failed!"`

File: dev-python/graphyte/graphyte-1.6.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1 python-r1
+
+DESCRIPTION="A small library to send data to graphite"
+HOMEPAGE="https://pypi.org/project/graphyte/"
+SRC_URI="https://github.com/Jetsetter/${PN}/archive/v${PV}.tar.gz"
```
Comment (alip) [inline dev-python/graphyte/graphyte-1.6.0.ebuild:11]: nitpick: it's good practise to include `${PN}` in tarballs using arrow notation.

File: dev-python/graphyte/graphyte-1.6.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="amd64 x86"
```
Comment (alip) [inline dev-python/graphyte/graphyte-1.6.0.ebuild:15]: why stable?

**Review by alip — APPROVED** [review]
looks good overall, I have a couple of minor qa bits which you  can fix as necessary on merge.

File: dev-python/graphyte/graphyte-1.6.0.ebuild (hunk 3 above)
Comment (65278) [reply dev-python/graphyte/graphyte-1.6.0.ebuild:15]: There's no way this is unstable.

File: dev-python/graphyte/graphyte-1.6.0.ebuild (hunk 2 above)
Comment (65278) [reply dev-python/graphyte/graphyte-1.6.0.ebuild:11]: Good catch. I simply forgot.

File: dev-python/graphyte/graphyte-1.6.0.ebuild (hunk 1 above)
Comment (65278) [reply dev-python/graphyte/graphyte-1.6.0.ebuild:27]: I really wanted to use eerror, so it stands.
